roles/reproducer/tasks/configure_post_deployment.yml

```diff
@@ -0,0 +1,27 @@
+---
+- name: Prepare scripts on controller-0
+  delegate_to: controller-0
```

Comment:
should not be "controllers"  group or we are sure that always would be controller-0 ?

Reply:
Afaik should always be controller-0. This is also reflected in other playbook and parts of cifmw.